indico/web/client/js/react/forms/fields.jsx

```diff
@@ -548,6 +548,23 @@ FinalRadio.propTypes = {
   label: PropTypes.string.isRequired,
 };
 
+const asciiSearch = (options, query) => {
+  // To find accented names in the drop-down easily
+  // the names (and query string as well) are transliterated
+  // to contain only ASCII characters.
+  // Example:
+  // * typing "turk" will show "Türkey" and "Turkmenistan"
+  // * typing "ürk" will show "Burkina Faso", "Türkey", and "Turkmenistan"
+  // See https://stackoverflow.com/a/37511463 for more details regarding the transliteration.
+  const transliterate = inputStr =>
+    inputStr
+      .toLowerCase()
+      .normalize('NFD')
+      .replace(/\p{Diacritic}/gu, '');
+  const asciiQuery = transliterate(query);
+  return options.filter(opt => transliterate(opt.text).includes(asciiQuery));
```

Comment:
I'd look through the places where we use FinalDropdown if we have any case where we pass something that's not just a String as the option's `text`, or places where we only have an element but no `text` (might be `undefined` in such a case)

Reply:
I've found only one place where no `text` in option: `TagModal`. It caused complete crash when started typing. 
However I think that component does not need this search functionality at all, so I just disabled it for that component. What do you think?

Reply:
I think it might be good to automatically use whatever fallback the default search uses in such cases.

Reply:
Well, I agree it is not too nice to hide the broken behavior with disabling the feature (which is reasonable regardless) :)
Since calling `toString()` seems meaningless the only option I can imagine to check whether `text` is `string`. Otherwise the unknown object is translated as empty string. Search feature works without useful result but at least does not break. 
Therefore I kept the search disabled for this `TagModal`.